src/scss/02_mixins/_link_mixins.scss

```diff
@@ -3,7 +3,7 @@
 
   &:focus,
   &:hover {
-    text-decoration: underline;
+    text-decoration: underline 2px;
```

Comment:
Also, should we use rem values (via `remify(2p)`) for these as the font is also defined in rem values to keep the text-decoration in proper ratio compared to the text in case user changes their default size?

Reply:
Fixed this with the magnificent regex that you created for me! Thank you 🙇